src/test/java/org/mule/test/http/internal/profiling/tracing/HttpCurrentSpanCustomizerTestCase.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.test.http.internal.profiling.tracing;
+
+
+import static org.mule.extension.http.api.HttpHeaders.Names.USER_AGENT;
+import static org.mule.extension.http.internal.listener.profiling.tracing.HttpListenerCurrentSpanCustomizer.HTTP_TARGET;
+import static org.mule.extension.http.internal.listener.profiling.tracing.HttpListenerCurrentSpanCustomizer.HTTP_USER_AGENT;
+import static org.mule.extension.http.internal.listener.profiling.tracing.HttpListenerCurrentSpanCustomizer.NET_HOST_NAME;
+import static org.mule.extension.http.internal.listener.profiling.tracing.HttpListenerCurrentSpanCustomizer.NET_HOST_PORT;
+import static org.mule.extension.http.internal.listener.profiling.tracing.HttpListenerCurrentSpanCustomizer.getHttpListenerCurrentSpanCustomizer;
+import static org.mule.extension.http.internal.request.profiling.tracing.HttpCurrentSpanCustomizer.HTTP_FLAVOR;
+import static org.mule.extension.http.internal.request.profiling.tracing.HttpCurrentSpanCustomizer.HTTP_METHOD;
+import static org.mule.extension.http.internal.request.profiling.tracing.HttpRequestCurrentSpanCustomizer.HTTP_URL;
+import static org.mule.extension.http.internal.request.profiling.tracing.HttpRequestCurrentSpanCustomizer.NET_PEER_NAME;
+import static org.mule.extension.http.internal.request.profiling.tracing.HttpRequestCurrentSpanCustomizer.NET_PEER_PORT;
+import static org.mule.extension.http.internal.request.profiling.tracing.HttpRequestCurrentSpanCustomizer.getHttpRequesterCurrentSpanCustomizer;
+import static org.mule.runtime.http.api.domain.HttpProtocol.HTTP_1_1;
+import static org.mule.test.http.AllureConstants.HttpFeature.HTTP_EXTENSION;
+
+import static org.eclipse.jetty.http.HttpMethod.GET;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.mule.extension.http.api.HttpRequestAttributes;
+import org.mule.extension.http.internal.request.profiling.tracing.HttpCurrentSpanCustomizer;
+import org.mule.runtime.api.util.MultiMap;
+import org.mule.runtime.http.api.domain.message.request.HttpRequest;
+import org.mule.sdk.api.runtime.source.DistributedTraceContextManager;
+
+import io.qameta.allure.Feature;
+import org.junit.Test;
+
+import java.net.URI;
+
+@Feature(HTTP_EXTENSION)
```

Comment:
Do we have a feature for tracing?
Also include Issue and Description annotations